draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1477,11 +1481,10 @@ JSON-SEQ serialization example:
       }
     }
 }
-<RS>{"time": 2, "name": "transport:packet_received", "data": { ... } }
-<RS>{"time": 7, "name": "http:frame_parsed", "data": { ... } }
+<RS>{"time": 2, "name": "transport:parameters_set", "data": { ... } }
+<RS>{"time": 7, "name": "transport:packet_sent", "data": { ... } }
```

Comment:
Why these changes?

Reply:
in this specific case, the newlines matter. While the format does support them, it might confuse readers. So I changed the events to ones that had a slightly shorter name but still seemed to be semantically valid.

Reply:
Ok.